stack.yaml

```diff
@@ -25,13 +25,13 @@ extra-deps:
 - servant-auth-server-0.4.0.0
 
 - git: https://github.com/serokell/auth-data-structures.git
-  commit: f9a0cd66f443d7b0e04a3c8ae8d475f8344b823b
+  commit: 6fb9e2dd1f3d951996b4e69e690a195551c060e2
```

Comment:
This is not master.

Reply:
Yes, its is not. Some of the code was moved to `auth-data-structures`, and I fixed some bugs and inconveniences there, so I changed the commit.

Reply:
The suggested way to solve this is to update `auth-data-structures` master.